dataland-frontend/src/components/forms/parts/fields/YesNoFormField.vue

```diff
@@ -59,10 +139,22 @@ export default defineComponent({
         No: "No",
       },
       isMounted: false,
+      qualityOptions: Object.values(QualityOptions).map((qualityOption: string) => ({
+        label: qualityOption,
+        value: qualityOption,
+      })),
+      currentValue: undefined,
```

Comment:
This should not be initialized as `undefined`, because then the computed property `isDataQualitiyRequired` potentially returns an undefined, even though its return type is `boolean`

Reply:
checking